.github/workflows/ci.yml

```diff
@@ -2,46 +2,64 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
     paths:
-      - '**.js'
-      - 'package.json'
-      - '.github/workflows/ci.yml'
+      - "**.js"
+      - "package.json"
+      - ".github/workflows/ci.yml"
 
 jobs:
   test:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        node: [14.x, 12.x, 10.x]
+        node: [">=14.3", "^12.17", engines]
```

Comment:
why are we hardcoding node versions?

Reply:
For some reason, they are not in sync with the actual latest releases. I particularly want at least 14.3 for #843. It's not much _harder_ than generic 14 or 12 actually as it still the semver range.

Reply:
Ahhhh ok 👍. I approve the merge

Reply:
@NotMoni great, hit the `Approval` button then 😉

Reply:
Oh I total forgot. I’m on mobile sorry